### ISSUE TITLE
[GitLab] only list projects with Merge Requests enabled

### ISSUE THREAD

**Issue by tmeijn** [lindell/multi-gitter#183]
**Describe the bug**
In GitLab a project can [disable Merge Requests](https://docs.gitlab.com/ee/user/project/settings/#sharing-and-permissions). Multi-Gitter currently does not exclude these projects.

**To Reproduce**
1. Have a group with a project that has Merge Requests disabled
2. Run `multi-gitter status` (did not try other commands)
3. Command fails with `GET https://gitlab.com/api/v4/projects/xxxxxxxxx/merge_requests: 403 {message: 403 Forbidden}`

**Expected behavior**
Multi-Gitter should exclude projects with Merge Requests disabled

**Additional context**
The [GitLab Groups API](https://docs.gitlab.com/ee/api/groups.html#list-a-groups-projects) has an option to only list projects with Merge Requests enabled.

In `go-gitlab` it's this option: https://github.com/xanzy/go-gitlab/blob/7859f4617e05e4d1e31afa4dffda24c6ea86147f/groups.go#L370:2

So I think the change to resolve this issue could be in [gitlab.go](https://github.com/lindell/multi-gitter/blob/master/internal/scm/gitlab/gitlab.go#L145:L155):

```diff
func (g *Gitlab) getGroupProjects(ctx context.Context, groupName string) ([]*gitlab.Project, error) {
	var allProjects []*gitlab.Project
	for i := 1; ; i++ {
		projects, _, err := g.glClient.Groups.ListGroupProjects(groupName, &gitlab.ListGroupProjectsOptions{
			ListOptions: gitlab.ListOptions{
				PerPage: 100,
				Page:    i,
			},
			IncludeSubgroups: &g.Config.IncludeSubgroups,
+                       WithMergeRequestsEnabled: true
		}, gitlab.WithContext(ctx))
		if err != nil {
```

**Comment by lindell** [comment]
That looks very reasonable 😄  Please create a PR and I'm happy to merge it.

**Comment by lindell** [comment]
Fixed with #184

**Comment by tmeijn** [comment]
Can confirm it's working with v0.33.1. Thanks again!
